static/src/js/components/MasterOverlayPanel/MasterOverlayPanel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const MasterOverlayPanel = (props) => {
+  const { children, panelHeight } = props
+
+  return (
+    <div className="search__panel" style={{ height: `${panelHeight}px`, willChange: 'height' }}>
```

Comment:
Inline styles? I thought that was bad?

Reply:
Moved the will-change to css. Need to keep the height so it can be dynamically updated.